app/static/css/themes/nord-light.css

```diff
@@ -0,0 +1,52 @@
+html {
+    background-color: #D8DEE9b3 !important;
+}
+
+body {
+    background-color: #f9fafb !important;
+}
+
+div {
+    /*background-color: #111 !important;*/
+    color: #3B4252 !important;
+}
+
+a:visited h3 div {
+    color: #A3BE8C !important;
+}
+
+a:link h3 div {
+    color: #4C566A !important;
+}
+
+a:link div {
+    color: #81a1c1 !important;
+}
+
+div span {
+    color: #3B4252 !important;
+}
+
+input {
+    background-color: #f9fafb !important;
+    color: #3B4252 !important;
+}
+
+#search-bar {
+    color: #4C566A !important;
+    background-color: #ffffff !important;
+}
+
+.search-container {
+    background-color: #f9fafb !important;
+}
+
+footer {
+    background-color: #f9fafb !important;
+    color: #3B4252;
+}
+
+.reskin {
+    background-color: #ffffff;
+    color: #3B4252;
+}
```

Comment:
I haven't looked through this extensively yet, but this light theme seems to have issues with div colors and/or text color:

![image](https://user-images.githubusercontent.com/33362396/95664573-379b2580-0b17-11eb-8db0-35afffc9212b.png)